README.md

```diff
@@ -1 +1,14 @@
-# ultimatejob
\ No newline at end of file
+#Job search engine:
+##UltimateJob.com
+
+####The goal is:
+allow people the ability to search for jobs over the internet directly from the hiring company
+
+####Description:
+a website that people will be able to sign in, set search criteria,
+and the system will find the matches from all the participating company sites.
+
+####How it works:
+1. The main page website that users will be able to sign in, and fill in their preferences.
+2. Database that contains all the users and their preferences + search history.
+3. n algorithm that searches over the internet (API) for a company career that establishes their jobs.
```

Comment:
3. A backend that searches participating companies career websites for search matches